examples/sum_views.cpp

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <beman/any_view/any_view.hpp>
+
+#include <iostream>
+#include <ranges>
+#include <vector>
+
+namespace bav = beman::any_view;
+using opt = bav::any_view_options;
+
+auto sum(bav::any_view<int, opt::input> v1, bav::any_view<int, opt::input> v2) {
```

Comment:
cool.

Reply:
```suggestion
auto sum(bav::any_view<const int> v1, bav::any_view<const int> v2) {
```
You don't need to specify `input` since that's already the default option. Also like `std::span`, it's idiomatic to const-qualify the element type if your intention isn't to modify the input.